lib/usermanager.js

```diff
@@ -53,6 +52,22 @@ exports = module.exports = {
     'DuplicateUserError': DuplicateUserError
   },
 
+  /**
+   * checks if a user is permitted the action on the resource
+   *
+   * @param {string} action
+   * @param {objectid} userId
+   * @param {objectid} tenantId
+   * @param {object} a content item
+   * @param {callback} next (function (err, isAllowed))
+   */
+
+  hasPermission: function(action, user, contentItem, next) {
+    permissions = require('./permissions');
```

Comment:
Without a rather large refactor, I think this is the only way to resolve the problem right now.